docs/config-schema.md

```diff
@@ -249,6 +250,18 @@ The **nested** `consul` config object has the following fields:
 
 
 
+## vault
+The **nested** `vault` config object has the following fields:
+
+
+
+| Config option | Required | Type | Description |
+| --- | --- | --- | --- |
+| `vaultAddr` | no | string | The Vault Address.  Can also be provided by the standard Vault envvar `VAULT_ADDR`.  This option takes priority over the envvar if provided. |
+| `vaultToken` | no | string | The vault token, can also be provided by it the standard Vault envvar `VAULT_TOKEN`.  This option takes priority over the envvar if provided. |
```

Comment:
Is this the standard precedence order for the agent?  In general, my expectations are cli arg > env var > file.  This way automated jobs can override on an environmental basis.

Reply:
Yes, if it is in the agent config file, it is authoritative.